mergify_engine/json.py

```diff
@@ -25,6 +25,7 @@ def register_type(enum_cls):
 class Encoder(json.JSONEncoder):
     def default(self, v):
         if isinstance(v, enum.Enum):
+            register_type(type(v))
```

Comment:
Again, the goal is not to register every type multiple times. Each type should be registered when declared.